WebViewControl/ResourcesManager.cs

```diff
@@ -9,6 +9,28 @@ namespace WebViewControl {
 
     public static class ResourcesManager {
 
+        private static readonly AssemblyCache cache = new AssemblyCache();
+
+        public class Resource {
+            public string Extension;
+            public Stream ResourceStream;
+
+            public Resource(string extension, Stream resourceStream) {
+                Extension = extension;
+                ResourceStream = resourceStream;
+            }
+        }
+
+        public static Resource LoadEmbeddedResource(Uri url) {
```

Comment:
1) Move this method after the TryGetResourceWithFullPath
2) Rename it to TryGetResource
3) Remove the Resource class and create an out param for the extension
4) Create an overload internal method without the out param

Reply:
we don't need the extension in the SS side though